Vagrantfile

```diff
@@ -2,11 +2,9 @@
 # vi: set ft=ruby :
 
 Vagrant.configure("2") do |config|
-  # vbox name
-  config.vm.box = "centos_6_5_64"
 
-  # vbox url
-  config.vm.box_url = "https://github.com/2creatives/vagrant-centos/releases/download/v6.5.1/centos65-x86_64-20131205.box"
```

Comment:
This box has problems I could not resolve. Guest additions would not install, causing the /vagrant dir not to mount, causing it to be useless.